context/Web3Modal.tsx

```diff
@@ -20,18 +21,18 @@ const metadata = {
 };
 
 export const config = createConfig({
-  chains: [PUB_CHAIN],
+  chains: [PUB_CHAIN, mainnet],
   ssr: true,
   transports: {
     [PUB_CHAIN.id]: http(PUB_WEB3_ENDPOINT, { batch: true }),
+    [mainnet.id]: http(PUB_WEB3_ENDPOINT, { batch: true }),
   },
   connectors: [
     walletConnect({
       projectId: PUB_WALLET_CONNECT_PROJECT_ID,
       metadata,
       showQrModal: false,
     }),
-    injected({ shimDisconnect: true }),
```

Comment:
Why don't we want to use the injected one?

Reply:
Nope, it comes by default with the Metamask id. If you add it, it shows metamask twice

Reply:
I see, nice catch then!